lib/article_json/export/apple_news/elements/embed.rb

```diff
@@ -0,0 +1,100 @@
+module ArticleJSON
+  module Export
+    module AppleNews
+      module Elements
+        class Embed < Base
+          # Embed
+          # @return [Hash]
+          def export
+            {
+              role: role,
+              URL: source_url,
+              caption: caption
+            }.compact
+          end
+
+          private
+
+          def role
+            @role ||=
+              case embed_type
+              when :youtube_video, :vimeo_video, :dailymotion_video
+                :embedwebvideo
+              when :facebook_video
+                :facebook_post
+              when :tweet
+                :tweet
+              when :slideshare
```

Comment:
Do we need `:slideshare` and `:soundcloud` cases here? Or does it mean they will be coming soon?